docs_src/microservices/general/index.md

```diff
@@ -84,3 +84,35 @@ All services have the ability to collect the following common service metrics
 - **SecuritySecretsStored** - Count of secret stored to the service's Secret Store.
 - **SecurityConsulTokensRequested** - Count of Consul tokens been requested.
 - **SecurityConsulTokenDuration** - Duration of obtaining Consul token.
+
+## URI for Files
+
+Different files like load configuration, units of measurements, device profiles, device definitions, and provision watches can be loaded either from the local file system or from a remote location.
+For the remote location, HTTP and HTTPS URIs are supported. When using HTTPS, certificate validation is performed using the system's built-in trust anchors.
+
+### Authentication
+
+#### username-password in URI (not recommended)
+
+Users can specify the username-password (`<username>:<password>@`) in the URI as plain text.
+This is ok network wise when using HTTPS, but if the credentials are specified in configuration or other service files, this is not a good practice to follow.
+
+!!! example "Example - configuration file with plain text `username-password` in URI"
+    ```
+        [UoM]
+        UoMFile = "https://myuser:mypassword@example.com/uom.yaml"
+    ```
+
+#### Secure Credentials (preferred)
+
+The `edgexSecretName` query parameter can be specified in the URI as a secure way for users to specify credentials.
+This parameter specifies a Secret Name from the service's Secret Store where the credentials reside. 
+
+!!! example "Example - configuration file with `edgexSecretName` query parameter"
+```
+[UoM]
+UoMFile = "https://example.com/uom.yaml?edgexSecretName=mySecretName"
+```
+
+The authentication type and credentials are contained in the secret data specified by the Secret Name. 
+Only `httpheader` is currently supported.
```

Comment:
Please expand on the expected elements in the secret data. ie.
```
type=httpheader
headername=<name>
headercontents=<contents>
```

Reply:
Probably also need to reference how to store the secrets. 
Secure Mode:
- https://docs.edgexfoundry.org/3.0/api/Ch-APIIntroduction/
   - the Secret API needs to be added to this
- https://docs.edgexfoundry.org/3.0/security/SeedingServiceSecrets/

Non-Secure Mode
- https://docs.edgexfoundry.org/3.0/microservices/configuration/CommonConfiguration/#common-configuration-properties
   - InsecureSecrets on Writable Tab ( another section that needs expanding... ;-) )

Reply:
I think the common config piece should go under the configuration files instead of the generic one here.

Reply:
It isn't about loading config, but about where the secrets for the URI are stored. Obviously we can't use InsecureSecrets when loading config from URI... :-)

Reply:
Created issues to address the SecretAPI comment and the InsecureSecrets comment
SecretAPI: #1196
InsecureSecrets Tab: #1197